src/slack/message/index.js

```diff
@@ -161,6 +242,13 @@ const getVolunteerHeading = (volunteers) => {
   return getSection(volunteerHeading);
 };
 
+/**
+ * Format volunteer section for slack.
+ *
+ * @param {Array} volunteers The volunteers to format the heading for.
+ * @param {Map} taskCounts A map of volunteers to the amount of their assigned tasks.
+ * @returns {object} The formatted volunteer section object.
```

Comment:
This one is actually returning an array of section objects from `getSection()`. Note that the `volunteerSections` variable is returned as-is. This variable just the return value of `volunteers.map()`. Array.prototype.map returns an array.